GVFS/GVFS.Common/FileBasedCollection.cs

```diff
@@ -390,6 +390,7 @@ private void WriteToDisk(string value)
                 throw new InvalidOperationException(nameof(this.WriteToDisk) + " requires that collectionAppendsDirectlyToFile be true");
             }
 
+            // TODO(Mac): Decide if we want to stick with \r\n on all platforms, move them all to \n, or use a mix

```

Comment:
Do we need to keep this TODO? As far as I know, both Windows and Mac are happy with this format as is.

Reply:
@sanoursa Good point, both platforms appear to be happy with it so we can take it out

Reply:
Resolved

Reply:
Resolved